addons/godot-xr-tools/xr/start_xr.gd

```diff
@@ -91,9 +90,7 @@ func _setup_for_openxr() -> bool:
 	print("OpenXR: Configuring interface")
 
 	# Set the render target size multiplier - must be done befor initializing interface
```

Comment:
I would also remove the `must be done before initializing interface` bit as initialising happens before any of our code is run. In Vulkan this actually fully works now. In OpenGL there is still an issue with releasing things because we're lacking dependency checking but the buffers aren't allocated until the first frame is rendered after `use_xr` is set.

Reply:
I've removed the comment.